content/hackathons/usw25/index.mdx

```diff
@@ -96,3 +96,12 @@ Run it using `qemu-system-x86_64` and find the flag.
 Upload the flag in the [registration form](https://forms.gle/sV3Vm9Fe73kg1gb37).
 
 Read the basic unikernel concepts [here](https://unikraft.org/docs/concepts).
+
+### Sessions
+
+#### Session 1 - Overview of Unikraft
+
+For the following sessions we will build and run applications using Unikraft.
+Unikraft supports multiple platforms (e.g. KVM, Firecracker, Xen), but we will focus on running applications using Qemu/KVM.
+
+Follow the steps under [the `Overview` guide](/guides/overview), and mark the items in [this spreadsheet](TODO).
```

Comment:
Let's create a Google form for each session. It will be used as attendance and as submission of results. Similar to the one we did for SSS](https://drive.google.com/drive/u/0/folders/1HTqPjsPC8yoce6zIJlDr_g-zRPAEIKP2). All participants will submit a short description of their work at the end of each session.

We will have 9 forms, for 9 sessions.